DragaliaAPI/Features/SavefileUpdate/V10Update.cs

```diff
@@ -0,0 +1,15 @@
+﻿using DragaliaAPI.Features.Emblem;
+using DragaliaAPI.Shared.Definitions.Enums;
+
+namespace DragaliaAPI.Features.SavefileUpdate;
+
+public class V10Update(IEmblemRepository emblemRepository) : ISavefileUpdate
+{
+    public int SavefileVersion => 10;
+
+    public async Task Apply()
+    {
+        if (!await emblemRepository.HasEmblem(Emblems.DragonbloodPrince))
```

Comment:
Could we also grant the user's currently equipped emblem?

Reply:
added